starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -52,7 +53,8 @@ public List<Inventory> findAll() {
    */
   public Inventory create(Inventory inventory) {
     // TODO
```

Comment:
Good job so far, but I believe you missed part of the instructions, it says to set the id to null.

Reply:
What is meant by the Mongo ID? The id of the collection I just put in?